src/Pumukit/EncoderBundle/Services/JobService.php

```diff
@@ -767,11 +767,16 @@ public function createTrackWithFile($pathFile, $profileName, MultimediaObject $m
      *
      * @return Track
      */
-    public function createTrack(MultimediaObject $multimediaObject, $pathEnd, $profileName, $language = null, $description = array())
+    public function createTrack(MultimediaObject $multimediaObject, $pathEnd, $profileName, $language = null, $description = array(), $pathFile = null)
     {
         $profile = $this->profileService->getProfile($profileName);
 
         $track = new Track();
+        if ($pathFile && $profile['master']) {
+            $pathInfo = pathinfo($pathFile, PATHINFO_FILENAME);
```

Comment:
```suggestion
            $pathInfo = pathinfo($pathFile, PATHINFO_BASENAME);
```

Use PATHINFO_BASENAME. Store the file extension is useful.